DataGenerator/DataGenerator.py

```diff
@@ -156,6 +161,22 @@ def SynchronizeData(config, subject_list):
 
     ## Download data
 
+
+def DataAugmentation(train_data, aug_transform):
+
```

Comment:
I'm not sure this is how we should do data augmentation. Transform should be simply passed to the argument maybe in the loop in the __getitem__ ? Would be good to discuss!